blue-note/theme.json

```diff
@@ -267,13 +276,56 @@
 		},
 		"elements": {
 			"button": {
+				"spacing": {
+					"padding": {
+						"top": "calc(0.667em + 1px)",
+						"right": "calc(1.333em + 1px)",
+						"bottom": "calc(0.667em + 1px)",
+						"left": "calc(1.333em + 1px)"
+					}
+				},
 				"border": {
-					"radius": "0"
+					"radius": "0",
+					"color": "transparent",
```

Comment:
Adding a transparent border here to avoid "jumping" on hover/focus/active states.